### PR TITLE
Output definitions

### PR DESCRIPTION
**Pull request by dustints**

### CODE REVIEW

File: lib/hammer_cli_gutterball/content_report.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        end
+      end
+    end
+
+    class ContentHostStatus < InfoCommand
+      include HammerCLIForemanTasks::Async
+      include ReportPrintable
+
+      action :system_status
+      command_name 'content-host-status'
```
Comment (komidore64) [inline lib/hammer_cli_gutterball/content_report.rb:24]: was hammer not properly translating `system` to `content-host` ? it's supposed to.

File: lib/hammer_cli_gutterball/content_report.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+          request = send_request
+          task_id = request.is_a?(Hash) ? request['id'] : request
+          task_progress(request)
+          taskoutput = load_task(task_id)
+          if taskoutput['output'] && taskoutput['output']['report_data']
+            print_data(taskoutput['output']['report_data'])
+          end
+          HammerCLI::EX_OK
+        end
+      end
```
Comment (komidore64) [inline lib/hammer_cli_gutterball/content_report.rb:16]: why is this execute override needed? i thought adding `include HammerCLIForemanTasks::Async` took care of the asynchronous bit.

Comment (dustints) [inline lib/hammer_cli_gutterball/content_report.rb:16]: for the insertion of `print_data`

Comment (dustints) [inline lib/hammer_cli_gutterball/content_report.rb:16]: i need to get at the task_id to load the task. there must be a better way to write this.

File: lib/hammer_cli_gutterball/content_report.rb (hunk 1 above)
Comment (komidore64) [inline lib/hammer_cli_gutterball/content_report.rb:24]: nevermind. not important. being explicit is good.

**Comment by komidore64** [comment]
building an RPM to make sure that `rpmbuild` doesn't blow up about anything.

**Comment by komidore64** [comment]
looks good

```
$ tito build --test --rpm --builder mock --arg mock=katello
WARNING: unable to lookup latest package tag, building untagged test project
WARNING: rel-eng/packages/rubygem-hammer_cli_gutterball doesn't exist in git, using current directory
WARNING: unable to lookup latest package tag, building untagged test project
WARNING: rel-eng/packages/rubygem-hammer_cli_gutterball doesn't exist in git, using current directory
Building package [rubygem-hammer_cli_gutterball-0.0.1-1]
Creating rpms for rubygem-hammer_cli_gutterball-git-0.59073bc in mock: katello

Wrote: /tmp/tito/rubygem-hammer_cli_gutterball-0.0.1-1.git.0.59073bc.el6.src.rpm

Using srpm: /tmp/tito/rubygem-hammer_cli_gutterball-0.0.1-1.git.0.59073bc.el6.src.rpm
Initializing mock...
Installing deps in mock...
Building RPMs in mock...

Wrote:
  /tmp/tito/rubygem-hammer_cli_gutterball-0.0.1-1.git.0.59073bc.el6.noarch.rpm
  /tmp/tito/rubygem-hammer_cli_gutterball-doc-0.0.1-1.git.0.59073bc.el6.noarch.rpm
```

ACK pending travis

**Pull request merged**